### PR TITLE
CP-2919-pause-fix added loop and pause

### PR DESCRIPTION
**Pull request by robertmudi**
JIRA Reference: [CP-3068]

### :memo: Description ️
Test fixes the device-drawer stress test. Now only device drawer is displayed instead of select device modal
-

### :muscle: Checklist before requesting a review - nice to have

- getState function in async thunk actions is correctly typed
- redux selectors are used in components / prop drilling is reduce

### :exclamation: Checklist before merging a pull request

- change went through the QA process
- translations are updated in dedicated application
- [CHANGELOG.md](./CHANGELOG.md) is updated

[CP-3068]: https://appnroll.atlassian.net/browse/CP-3068?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dkarski — APPROVED** [review]

File: apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts [hunk 1]
```diff
@@ -60,6 +61,11 @@ describe("Kompakt switching devices", () => {
       })
 
       await browser.pause(6000)
+
+      // Add a 4-second wait after adding the first device
+      if (i === 0) {
```
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts:66]:
```suggestion     

      // Add a 4-second wait after adding the first device
      if (i === 0) {
        await browser.pause(10000)
      } 
     else {
      await browser.pause(6000)
      }
```

**Review by tomaszmaleckimudita — CHANGES_REQUESTED** [review]

File: apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts [hunk 2]
```diff
@@ -60,6 +61,11 @@ describe("Kompakt switching devices", () => {
...
+        await browser.pause(4000)
+      }
```
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts:68]:
```suggestion
      // Add a 4-second wait after adding the first device
      if (i === 0) {
        await browser.pause(10000)
      } else {
      await browser.pause(6000)
      }
```

**Review by tomaszmaleckimudita — CHANGES_REQUESTED** [review]

File: apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts (hunk 2 above)
Comment (robertmudi) [reply apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts]: simplified the test

**Review by dkarski — APPROVED** [review]

**Review by tomaszmaleckimudita — APPROVED** [review]
Works ok

**Pull request merged**